cmd/vela-cli/load.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"github.com/go-vela/cli/action/config"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/urfave/cli/v2"
+)
+
+// load is a helper function that loads the necessary configuration for the CLI.
+func load(c *cli.Context) error {
```

Comment:
Here is where we moved the `go-vela/cli/cmd/vela-cli/load()` function to a separate file